src/s2wasm.h

```diff
@@ -35,15 +33,17 @@ cashew::IString EMSCRIPTEN_ASM_CONST("emscripten_asm_const");
 class S2WasmBuilder {
   AllocatingModule& wasm;
   MixedArena& allocator;
-  char *s;
+  const char *s;
+  bool debug;
 
 public:
-  S2WasmBuilder(AllocatingModule& wasm, char *input) : wasm(wasm), allocator(wasm.allocator) {
-    s = input;
-    scan();
-    s = input;
-    process();
-    fix();
+ S2WasmBuilder(AllocatingModule& wasm, const char* input, bool debug)
```

Comment:
clang-format. Do you have a style guide I can teach it? I'd rather just blindly format stuff, we can tell if which style to follow.